scenarios/glue_privesc/terraform/variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "profile" {
+  description = "The AWS profile to use."
+  type        = string
+}
+
+variable "region" {
+  description = "The AWS region to deploy resources to."
+  default     = "us-east-1"
+  type        = string
+}
+
+variable "cgid" {
+  description = "CGID variable for unique naming."
+  type        = string
+}
+
+variable "cg_whitelist" {
+  default = "../whitelist.txt"
```

Comment:
Remove default its wrong, the python script passes in a list(strings).

There is an [open PR](https://github.com/RhinoSecurityLabs/cloudgoat/pull/226/files#diff-6e3c59b2d3b34ccfaf2d076239de4ae66a9104991a0996d664bf6cda39b3cc0aR22) to fix the default value

> This default value is not compatible with the variable's type constraint: list of string required.